pytest.ini

```diff
@@ -0,0 +1,11 @@
+[pytest]
+
+addopts = -v --junit-xml=unittests.xml -s  --strict-markers --strict
+
+markers =
```

Comment:
why we need these markers?.

Reply:
actually it was intended for `submodule`, removing from this project